### ISSUE TITLE
Build not possible without pdf of infographic

### ISSUE THREAD

**Issue by katkolouchova** [faktaoklimatu/web-core#143]
It would be useful to be able to generate a preview without having to upload pdf of infographic (in case the text is finished first). Or to return a better error message of why the build failed.

**Comment by mukrop** [comment]
Well, depends.

The point of tests is to check that all linked media exists (apart from checking other things). Allowing build without the infographic would mean having an exception to this rule (i.e. effectively not checking if the infographic was uploaded or not).

Though, on the other side, I understand that uploading a placeholder effectively renders that check useless as well :-/.

What do you think? Any opinions from @mgrabovsky?

**Comment by mgrabovsky** [comment]
I agree that it would be a useful feature to have. Sometimes, we might just want to check what the text looks like on the page without having to export the infographic or running the build locally.

Alternatively, can we just disable the link checks for preview branches and only require them for merging pull requests?
